source/index.erb

```diff
@@ -2,36 +2,53 @@
 title: Energy Globe Award
 ---
 
-<div id="application">
-
-  <header>
-    <h1 class="headline">Energy Globe Award</h1>
-  </header>
-
-  <section ng-controller="AwardsController">
-
-    <section class="filters">
-      <h2>Kategorien</h2>
-      <ul class="category-filter" ng-repeat="category in filters.category.names">
-        <li>
-          <input type="checkbox" id="category_checkbox_{{category}}" ng-model="filters.category.filterMap[category]">
-          <label for="category_checkbox_{{category}}">{{category}}</label>
-        </li>
-      </ul>
-
-      <h2>Jahre</h2>
-      <ul class="year-filter" ng-repeat="year in filters.year.names">
-        <li>
-          <input type="checkbox" id="year_checkbox_{{year}}" ng-model="filters.year.filterMap[year]">
-          <label for="year_checkbox_{{year}}">{{year}}</label>
-        </li>
-      <ul>
-    </section>
-
-    <div class="award" ng-repeat="award in filteredAwards">
-      <h3>{{ award.title }}</h3>
+<header class="page-header">
+  <div class="page-header-container">
+    <div class="page-header-content">
+      <h1 class="page-header-headline">Energy Globe Award</h1>
```

Comment:
Sure :-) I have read that book, and its pretty much what I told you just now. Only in cleaner terminology. My opinion was also very much influenced by it.